src/include/miopen/reduce/problem_description.hpp

```diff
@@ -25,6 +25,7 @@
  *******************************************************************************/
 #pragma once
 
+#include "miopen/miopen.h"
```

Comment:
this lib seems to be automatically added by the VSCode extension, it seems not necessary to include this lib, rm it

Reply:
resolved